src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random randomColor = new Random();
+
+    public static String getRandomColor() {
+        return Colors.values()[randomColor.nextInt(Colors.values().length)].toString();
+    }
```

Comment:
```suggestion
    private final Random randomColor = new Random();

    public String getRandomColor() {
        return Colors.values()[randomColor.nextInt(Colors.values().length)].toString();
    }
```

давай сделаем метод getRandomColor() не статическим (https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5)